packages/dd-trace/src/guardrails/index.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
```

Comment:
This file was copied as-is with only ECMAScript version related changes, as well as calling a callback instead of initializing directly.